docker-compose.yml

```diff
@@ -1,10 +1,24 @@
 version: '2'
 services:
+  mysql:
+    image: mysql:5.7
+    ports:
+      - "3306:3306"
+    environment:
+      MYSQL_USER: sharetribe
+      MYSQL_PASSWORD: secret
+      MYSQL_DATABASE: sharetribe_production
+      MYSQL_ROOT_PASSWORD: secret
+
   web:
     build: .
     ports:
       - "3000:3000"
     env_file: .env
+    links:
+      - mysql:mysql
+    depends_on:
+      - mysql
```

Comment:
Pretty sure you can use either `links` or `depends_on`, but both are not needed.

https://stackoverflow.com/questions/35832095/difference-between-links-and-depends-on-in-docker-compose-yml